gammapy/spectrum/core.py

```diff
@@ -269,6 +269,19 @@ def to_table(self):
         else:
             meta.update(hduclas2='BKG', )
 
+        # Add general optional keywords if the member exists rather than default value. LBYL approach
```

Comment:
Ah sorry I just see the difference - you don't want to give default value, so maybe ignore comment 2)